test/conformance/api/v1alpha1/domain_mapping_test.go

```diff
@@ -0,0 +1,100 @@
+// +build e2e
+
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+	"net/url"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	duckv1 "knative.dev/pkg/apis/duck/v1"
+	"knative.dev/serving/pkg/apis/serving/v1alpha1"
+	"knative.dev/serving/test"
+	"knative.dev/serving/test/conformance/api/shared"
+	v1test "knative.dev/serving/test/v1"
+)
+
+func TestDomainMapping(t *testing.T) {
+	if !test.ServingFlags.EnableAlphaFeatures {
+		t.Skip("Alpha features not enabled")
+	}
```

Comment:
We should enable this in `.github/workflows/kind-e2e.yaml` and `./test/e2e-tests.sh` otherwise nothing is running this.  I feel like we should be running this at least somewhere.

I'd kinda like to see the istio class annotation moved to be config-based before we start adding e2e tests that will light up folks testgrid.

Reply:
added to kind-e2e in https://github.com/knative/serving/pull/9837, Im gonna ping on slack to try to get some hand-holding for how to go about trying to iterate on modifications to e2e-tests.sh :)

Reply:
.. and added prow e2e in https://github.com/knative/serving/pull/9851. Think this may be ready to go now.